WorkloadGenerator/config.py

```diff
@@ -8,3 +8,7 @@
 AUDIT_SERVER_NAME = 'AuditService'
 AUDIT_SERVER_IP = 'localhost'
 AUDIT_SERVER_PORT = 6500
+
+TRIGGER_SERVER_NAME = 'TriggerService'
+TRIGGER_SERVER_IP = 'localhost'

```

Comment:
Shouldn't we also be setting the IP addresses to the container names? I don't think this will work otherwise. Is this something you envision us changing later?

Reply:
I am talking here strictly about external services from each service's perspective. I.e., transaction service should run on its own localhost but it should query the actual IP (non-localhost) for other services. Even then, the config file can be changed to contain absolute IPs but each service can use it only for other services while resorting to default for its own.

Reply:
hmm ya I guess I just had it running on localhost for now cause I was not running the api's as part of the docker containers yet and just running them directly